core/cpu.c

```diff
@@ -222,6 +222,7 @@ bool vcpu_is_panic(struct vcpu_t *vcpu)
     if (vcpu->panicked) {
         hax_error("vcpu has panicked, id:%d\n", vcpu->vcpu_id);
         hax_panic_log(vcpu);
+		htun->_exit_reason = vmx(vcpu, exit_reason).basic_reason;
```

Comment:
use spaces
```suggestion
        htun->_exit_reason = vmx(vcpu, exit_reason).basic_reason;
```

Reply:
Sorry, hopefully this was the last tab vs. space issue, I did a grep on \t now and didn't find any more areas